axelor-contract/src/main/resources/views/ContractLine.xml

```diff
@@ -56,6 +75,21 @@
     </panel-tabs>
   </form>
 
+  <action-group name="action-contract-line-group-onload">
+    <action name="action-contract-line-attrs-hide-consumption-line"/>
+    <action name="action-contract-line-method-hide-axis-analytic-account"/>
+    <action name="action-contract-line-method-print-analytic-account"/>
+  </action-group>
+
+  <action-group name="action-contract-line-group-analytic-template-onchange">
+    <action name="action-contract-line-method-create-distribution"/>
+    <action name="action-contract-line-method-print-analytic-account"/>
+  </action-group>
+
+  <action-group name="action-contract-line-group-analytic-axis-onchange">
+    <action name="action-contract-line-method-axis-create-analytic-line"/>
+  </action-group>
```

Comment:
why do you need to chain all these action-method ? there should be less actions and more different controller methods and service methods.